app/src/main/java/org/fossasia/openevent/general/order/OrdersUnderUserFragment.kt

```diff
@@ -76,6 +76,9 @@ class OrdersUnderUserFragment : Fragment(), BottomIconDoubleClick {
         ordersUnderUserVM.connection
             .nonNull()
             .observe(viewLifecycleOwner, Observer { isConnected ->
+                if (!isConnected) {
+                    showNoInternetScreen(true)
+                } else {
                 val currentItems = ordersUnderUserVM.eventAndOrderPaged.value
```

Comment:
Still wrong. Do you really not know what indentation should a block inside {} should have?

Reply:
I did not. Now I know sir.